Storage.md

```diff
@@ -0,0 +1,261 @@
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    prefix: "FullScreenShenanigans"
+});
+```
+
+## Getters and Setters
+
+ItemsHoldr has a number of functions that return information about the collection of items or a specific item.
+
+```typescript
+// simple gets
+ItemsHolder.getKeys();
+ItemsHolder.getValues();
+ItemsHolder.exportItems();
+ItemsHolder.getAutoSave();
+ItemsHolder.getLocalStorage();
+ItemsHolder.getPrefix();
+ItemsHolder.getContainer();
+ItemsHolder.getDisplayChanges();
+
+// returns the indexed key
+ItemsHolder.key(1);
+// returns the value under the key
+ItemsHolder.getItem("color");
+// returns whether the key is in the collection
+ItemsHolder.hasKey("color");
+```
+
+To add an item to the collection, use `addItem`.
+
+```typescript
+ItemsHolder.addItem("color", { value: "red" });
+```
+
+To remove an item, use `removeItem`.
+
+```typescript
+ItemsHolder.removeItem("color");
+```
+
+`setItem` is used to set the value of an item.
+
+```typescript
+ItemsHolder.setItem("color", "purple");
+```
+
+To manually update a changed item's value in localStorage, use `saveItem`.
+
+```typescript
+ItemsHolder.saveItem("color");
+```
+
+To save all the items' values in localStorage, use `saveAll`
+
+```typescript
+ItemsHolder.saveAll();
+```
+
+`increase` and `decrease` add and subtract respectively from the item's value.
+
+```typescript
+ItemsHolder.increase("weight", 14);
+ItemsHolder.decrease("weight", 10);
+```
+
+`toggle` flips the boolean value of the item.
+
+```typescript
+ItemsHolder.toggle("started");
+```
+
+`checkExistence` can be used to check if there is an item under the specified key, and if not, add it to the collection.
+
+```typescript
+ItemsHolder.checkExistence("color");
+```
+
+By default if an item does not exist when `setItem`, `getItem`, `increase`, or `toggle` are called, the item is then added to the collection in the process of the respective function.
+Define `allowNewItems` to explicitly enable or disable this feature.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ allowNewItems: false });
+```
+
+## Updating
+
+When an item's value changes, the module checks its triggers, modularity, and element status.
+
+Triggers are functions that get run when an item's value equals the specified value.
+
+```typescript
+ItemsHolder.addItem("color", {
+    triggers: {
+        red: () => { this.value = "black"; }
+    }
+});
+ItemsHolder.setItem("color", red);
+```
+
+Modularity restricts an item's value to a range from 0 to the specified max and runs the `onModular` function as many times until the value hits the modular range.
+
+```typescript
+ItemsHolder.addItem("time", {
+    modularity: 12,
+    num: 0,
+    onModular: () => { this.num += 1; }
+});
+ItemsHolder.setItem("time", 100);
+```
+
+If the item is an HTML element, the value is also updated on the page.
```

Comment:
Put this in its own section. The HTML display is separate functionality from the rest of the module.

You may also want to note that it will be split off into a separate module some time in the future.